.npmrc

```diff
@@ -1 +1,2 @@
 unsafe-perm=true
+legacy-peer-deps=true
```

Comment:
I was discussing this with @timotheeg on this, I think we can manual key in this flag as it is not supposed to be a permanent fixture in our development process.

The assumption is that `--legacy-peer-deps` is not required after angular is exterminated.

Reply:
That's fair, I think it's not necessary but mostly just a temporary DX thing. I'm ok to get rid of it, nbd haha.

For context though: I'm not confident that we can get rid of it along with angular, we may end up needing to do some dependency cleanup in a separate PR. The flag was introduced because a lot of angular packages violated the new resolution strategy, but other packages in backend/react also did. I don't know the extent of how easy it will be to upgrade everything so we can get rid of the flag, which is why I thought we might as well add it first, and then remove it later along with this ticket https://github.com/opengovsg/FormSG/issues/5362.

If you still feel we can leave it out from this PR I have no issues removing it!

Reply:
I see, I think tracking this from another issue works too. I, too, am not sure if other packages are affected as well. Then it makes sense to keep this change as it is due to `npm 8` upgrade.